test/test_tv_tensors.py

```diff
@@ -305,3 +305,16 @@ def test_set_return_type():
     assert type(img + 3) is tv_tensors.Image
 
     tv_tensors.set_return_type("tensor")
+
+
+def test_return_type_input():
+    img = make_image()
+
+    # Case-insensitive
+    with tv_tensors.set_return_type("tvtensor"):
+        assert type(img + 3) is tv_tensors.Image
+
+    with pytest.raises(ValueError, match="return_type must be"):
+        tv_tensors.set_return_type("typo")
+
+    tv_tensors.set_return_type("tensor")
```

Comment:
We do have a global setup teardown fixture to prevent any of this leaking, right? If so why set it to tensor and then do nothing with it?

Reply:
Yeah, we don't need it.

This follows up to https://github.com/pytorch/vision/pull/7828#discussion_r1293636144